AdblockPlusSafari/AdblockPlusExtras.m

```diff
@@ -189,20 +205,6 @@ - (void)setNeedsDisplayErrorDialog:(BOOL)needsDisplayErrorDialog
   [self.adblockPlusDetails synchronize];
 }
 
-- (BOOL)anyLastUpdateFailed
```

Comment:
It looks as if this function is just moved down but doesn't actually change. Is that related to issue 3184?